samplingsimulatorpy/generate_virtual_pop.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import pandas as pd
+
+def generate_virtual_pop(size, distribution_func, *para):
+
+    """
+    Create a virtual population
+
+    Parameters
+    ----------
+    size : int
+        The size of the virtual population 
+    distribution_func : func
+        The function that 
+    *para : int
+        The parameters the distribution_func is using
+
+    Returns
+    -------
+    pd.DataFrame
+        The virtual population as a dataframe
+
```

Comment:
Add Raises section to docstring, see example below

```suggestion
    Raises
    -------
    ValueError
        size input is a positive integer
```

Reply:
Nice catch! Addressed #38